data/applications/starburst.yaml

```diff
@@ -15,7 +15,7 @@ spec:
   support: third party support
   quickStart: ''
   getStartedLink: https://www.starburst.io/platform/starburst-galaxy/
-  beta: true
+  beta: false
   betaText: |-
```

Comment:
Optional: remove `betaText` as well

Reply:
@taneem-ibrahim @jeff-phillips-18 is beta text automatically hidden when `beta: false` ?

Reply:
@bdattoma yes